docker/env/Dockerfile

```diff
@@ -21,9 +21,9 @@ RUN mkdir /build && \
     pip3 install -r requirements.txt --root=/build --prefix=./ --ignore-installed --no-warn-script-location
 
 FROM python:$PYTHON_IMAGE_TAG
-ARG KUBECTL_VERSION=1.27.3
-ARG EKSCTL_VERSION=0.146.0
-ARG HELM_VERSION=3.12.1
+ARG KUBECTL_VERSION=1.28.4
```

Comment:
this is something we should be doing in tandm of update the k8s we are using in the tests
kubectl should be backward compatible, but it still might break our code.

we usually advance it, when we need specific features out of it.